api/app/signals/settings/development.py

```diff
@@ -52,3 +52,20 @@ def show_toolbar(request):
     'API_SEARCH_ENABLED': True,
     'SEARCH_BUILD_INDEX': True,
 }
+
+# Set console logging to DEBUG
+LOGGING['handlers'].update({  # noqa F405
+    'console': {
+        'level': 'DEBUG',
+        'class': 'logging.StreamHandler',
+        'formatter': 'console',
+    },
+})
+
+# Log queries to the console
```

Comment:
This gets a bit noisy, but ok.

Reply:
That's why it is in development. It is a lot of noise but it performs better than the Django debug toolbar. Also it does not kill your docker container like the Django debug toolbar does in some cases.